docs/deployment.md

```diff
@@ -3,7 +3,7 @@
 
 > [!NOTE]
 > **Before you start**
-> This guide assumes you have created a Catalyst storefront using [the CLI](/cli) or the [Advanced configuration guide](/monorepo). Your project directory should contain only a Next.js application rather than the monorepo in the [Catalyst GitHub repository](https://github.com/bigcommerce/catalyst). If you want to develop using the monorepo, you must modify these instructions. For more information, see [Monorepo](/monorepo).
+> This guide assumes you have created a Catalyst storefront using [the CLI](/docs/getting_started.md) or the [Advanced configuration guide](/docs/monorepo.md). Your project directory should contain only a Next.js application rather than the monorepo in the [Catalyst GitHub repository](https://github.com/bigcommerce/catalyst). If you want to develop using the monorepo, you must modify these instructions. For more information, see [Monorepo](/docs/monorepo.md).
```

Comment:
@christensenep For the CLI link, I noticed that the filename is `cli.md` in github, so the link might need to be `/docs/cli.md`